src/components/questionTypes/Free/creation/Preview.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+
+type Props = {
+  description: string,
+  title: string,
+}
+
+const defaultProps = {
+  description: 'DESCRIPTION',
+  options: {
+    restrictions: {
+      max: null,
+      min: null,
+      type: 'NONE',
+    },
+  },
+  title: 'TITLE',
+}
+
+const Preview = ({ title, options, description }: Props) => (
+  <div className="preview">
+    <div className="title">{title || 'TITLE'}</div>
+    <div className="description">{description || 'DESCRIPTION'}</div>
+    {
+      options.restrictions.type === 'NUMBERS' && options.restrictions.min && options.restrictions.max &&
```

Comment:
@pascalzehnder If type is NUMBERS and min and max are not set, display a simple input field for an arbitrary number (basically another type of restriction).